components/encryption/src/metrics.rs

```diff
@@ -15,12 +15,12 @@ lazy_static::lazy_static! {
         "tikv_encryption_is_initialized",
         "Flag to indicate if KeyDictionary encryption is initialized"
     ).unwrap();
-    pub static ref ENCRYPT_DECRPTION_FILE_HISTOGRAM: HistogramVec = register_histogram_vec!(
-        "tikv_encryption_write_read_file_duration_seconds",
+    pub static ref ENCRYPT_DECRPTION_FILE_GAUGE: GaugeVec = register_gauge_vec!(
+        "tikv_encryption_write_read_file_duration_micros",
```

Comment:
Why choose micros? It seems inconsistent with other metrics.

Reply:
@wujy-cs PTAL

Reply:
The metrics is measuring a whole file read/write. What do you suggests?

Reply:
I suggest to use second as unit just like other metrics does.

Reply:
Didn't realize metrics are float instead of integer.